transport/ssl/src/main/java/org/kaazing/gateway/transport/ssl/SslConnector.java

```diff
@@ -124,7 +124,7 @@ public void setSecurityContext(SecurityContext securityContext) {
             // avoid caching SSLSession in shared SSLContextFactory instance
             // Note: SSLSessionContext.setSessionCacheSize(0) means unlimited,
             // so we use 1 instead
-            sslContextFactory.setServerSessionCacheSize(1);
+            sslContextFactory.setClientSessionCacheSize(1);
```

Comment:
It is now here: https://github.com/kaazing/gateway/pull/813.  I'll change this PR request with the other issues to remove the TLS fix